docs/feature-guides/clarinet-js-sdk.md

```diff
@@ -1,10 +1,11 @@
 # Clarinet SDK
 
-The Clarinet SDK is used to spawn and interact with a Simnet from Node.js.
+The Clarinet SDK is a JavaScript library to spawn and interact with a simulated Clarinet environment, also known as simnet.
 
 A Simnet is a simulated network that simulates the Stacks blockchain and runs the Clarity VM, without the need for actual Stacks and Bitcoin nodes (unlike Devnet, Testnet and Mainnet).
 
 Here is a non-exhaustive list of some of the use-cass:
+
 - Call public and read-only functions from smart contracts
 - Get clarity maps or data-var values
 - Get contract ABI
```

Comment:
The references to "ABI" may not be a common convention to some developers. Perhaps worth expanding on the acronym somewhere in the page.

Reply:
Made two suggestions too fully remove ABI. I don't it's 100% technically correct anyway